### ISSUE TITLE
output assigned to `ccf_var` could be a 1d or 2d array

### ISSUE THREAD

**Issue by hematthi** [RvSpectML/EchelleCCFs.jl#57]
https://github.com/RvSpectML/EchelleCCFs.jl/blob/a639217e24ebc59282d42e27d739d3aea1a032c6/src/ccf/calc_ccf.jl#L291-L293

Depending on whether `calc_ccf_covar` is true or false, `ccf_covar_out` could be 1d or 2d

**Comment by eford** [comment]
True.  It probably would be better to use a diagonal matrix when calc_ccf_covar=false.  But we'd need to make sure that won't break things elsewhere.
